yb-voyager/src/callhome/diagnostics.go

```diff
@@ -264,6 +266,22 @@ type ImportDataPhasePayload struct {
 	ControlPlaneType    string `json:"control_plane_type"`
 }
 
+type YBClusterMetrics struct {
+	Timestamp time.Time    `json:"timestamp"`   // time when the metrics were collected
+	CPUAvgPct float64      `json:"cpu_avg_pct"` // mean of node CPU% across all nodes
+	MemAvgPct float64      `json:"mem_avg_pct"` // mean of node Mem% across all nodes
+	Nodes     []NodeMetric `json:"nodes"`       // one entry per node
+}
+
+// per-node snapshot
+type NodeMetric struct {
```

Comment:
can we also add `memory_free`, `memory_total`, `memory_available` to this struct? 
Adaptive parallelism also does throttles based on this.

Reply:
done